packages/embedded-wallet-service-wagmi/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@paperxyz/embedded-wallet-service-wagmi",
+  "description": "Wagmi connector for Embedded Wallets by Paper",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
+  "keywords": [
+    "embedded wallets",
+    "crypto",
+    "wallet",
+    "paper sdk",
+    "crypto wallet",
+    "wagmi"
+  ],
+  "bugs": {
+    "url": "https://github.com/paperxyz/js-sdk/issues"
+  },
+  "homepage": "https://github.com/paperxyz/js-sdk",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/paperxyz/js-sdk.git"
+  },
+  "author": "Paper <support@withpaper.com>",
+  "scripts": {
+    "lint:fix": "yarn lint --fix",
+    "lint": "eslint \"src/**/*.ts*\"",
+    "build": "tsup src/index.ts --env.NODE_ENV production --env.SDK_VERSION $npm_package_version",
+    "clean": "rm -rf .turbo node_modules dist",
+    "prepublish": "yarn build",
```

Comment:
we don't need this and the line below, this is handled by the `package.json` defined in the tld